bindata/network/ovn-kubernetes/managed/004-config.yaml

```diff
@@ -52,6 +52,9 @@ data:
 {{- if .DNS_NAME_RESOLVER_ENABLE }}
     enable-dns-name-resolver=true
 {{- end }}
+{{- if .OVN_PERSISTENT_IPS_ENABLE }}
+    enable-persistent-ips=true
+{{- end }}
```

Comment:
If we pass the option as parameter (which we need to trigger a rollout) then I don't think we usually add the option in the config map so I wouldn't do it here either.

Reply:
OK I now realize why I was adding it here ... It was granting me a simple way to unit test the feature enablement.

Can I do something similar w/ the rollout parameter ? 

Are there tests doing that ?

Reply:
I don't see any that go as deep as verifying the arguments, I wonder why, they should not be complicated to make.

I also see a lot of repetition in the config map and the arguments now that I checked, so I guess it is not as clear cut as I thought. However testing the config map does not really cover up for testing the arguments 100%

Reply:
I've updated the tests to assure we install the CRD.

Thanks for raising it.

Reply:
Done.